src/linux_network.py

```diff
@@ -2,77 +2,153 @@
 #
 # igcollect - Linux network
 #
-# Copyright (c) 2016, InnoGames GmbH
+# Copyright (c) 2018, InnoGames GmbH
 #
 
 from argparse import ArgumentParser
 from time import time
+import os
+
+
+class InterfaceStatistics(object):
+    '''
+        Supported types of interfaces for a metrics sending
+        If there is more than one conditions they multipied by AND
+    '''
+    NET_TYPES = {
+        'bond': {'bonding': '_check_dir'},
+        'bond_slave': {'bonding_slave': '_check_dir'},
+        'bridge': {'bridge': '_check_dir'},
+        'bridge_slave': {'brport': '_check_dir'},
+        'general_slave': {'master': '_check_symlink'},
+        'lo': {("772", ): "_check_type"},
+        'ovs-br0': {'br0': '_check_name'},
+        'ovs-system': {'ovs-system': '_check_name'},
+        'phys': {'device': '_check_symlink'},
+        'tunnel': {("768", "776"): '_check_type'},
+        'vlan': {"DEVTYPE=vlan": '_check_uevent'},
+    }
+    _scn = '/sys/class/net'
+
+    def __init__(self, included_types=[]):
+        self.included_types = included_types
+        self.netdev_stat = {}
+
+    def _check_dir(self, dev, directory):
+        return os.path.isdir(os.path.join(self._scn, dev, directory))
+
+    def _check_name(self, dev, name):
+        return dev == name
+
+    def _check_symlink(self, dev, symlink):
+        return os.path.islink(os.path.join(self._scn, dev, symlink))
+
+    def _check_type(self, dev, types):
+        _is_tunnel = False
```

Comment:
There is no need to prefix variables in Python.  They are always in local scope.